### PR TITLE
Allow quotes when typing 'delete'

### PR DESCRIPTION
**Pull request by lukaw3d**
since our instructions are `To confirm and proceed, please type 'delete' below.`

### CODE REVIEW

File: src/app/components/DeleteInputForm/index.tsx [hunk 1]
```diff
@@ -21,7 +21,9 @@ export function DeleteInputForm({ children, onCancel, onConfirm }: DeleteInputFo
       <FormField
         name="type_delete"
         validate={(value: string | undefined) =>
-          !value || value.toLowerCase() !== t('deleteForm.confirmationKeyword', 'delete').toLowerCase()
+          !value ||
+          value.toLowerCase().replace(/['"“”]/g, '') !==
```
Comment (lubej) [inline src/app/components/DeleteInputForm/index.tsx:25]: Doesn't this enable you to write `'''''''''delete'''''''''`?

**Review by lubej — APPROVED** [review]

File: src/app/components/DeleteInputForm/index.tsx (hunk 1 above)
Comment (lukaw3d) [reply src/app/components/DeleteInputForm/index.tsx:25]: yeah :shrug: still seems fine

Comment (lubej) [reply src/app/components/DeleteInputForm/index.tsx:25]: Also `'''''d'e''l'e''t'e'''''` would be possible. But I guess it doesn't matter in the end.

Comment (lubej) [reply src/app/components/DeleteInputForm/index.tsx:25]: I would personally rather see we remove apostrophes entirely. And mark the text different(italic, bold), that is supposed to be written down in confirmation dialog.

Comment (lukaw3d) [reply src/app/components/DeleteInputForm/index.tsx:25]: good idea
https://github.com/oasisprotocol/oasis-wallet-web/pull/1953

**Pull request closed without merging**